pkg/announce/options.go

```diff
@@ -69,3 +72,8 @@ func (o *Options) WithChangelogFile(changelogFile string) *Options {
 	o.changelogFile = changelogFile
 	return o
 }
+
+func (o *Options) WithGoVersion(goVersion string) *Options {
+	o.goVersion = goVersion
+	return o
+}
```

Comment:
Do we want to add the version always? I don't see a strong reason for making it optional.

Reply:
I did it because of the consistency. We pass all other variables via Options struct, so I thought it would eventually make sense to do the same for the Go version.

However, the Go version is not really dynamic, unlike other properties, so maybe I should just do it from `pkg/announce/announce.go` instead of passing it as an option.